pkg/controller.v1beta1/suggestion/suggestion_controller_test.go

```diff
@@ -19,6 +19,9 @@ package suggestion
 import (
 	"context"
 	"encoding/json"
+	"github.com/kubeflow/katib/pkg/controller.v1beta1/util"
```

Comment:
Can we move the katib deps to the third group and the third party deps to the second group?

Reply:
It makes sense.
I moved it!